src/App.jsx

```diff
@@ -14,70 +16,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
```

Comment:
```suggestion
  const [selectedGood, setselectedGood] = useState('Jam');
``` that's was naming of variables from tasks)

Reply:
that's was naming of variables from tasks)